cmd/server/main.go

```diff
@@ -138,6 +138,7 @@ func buildCLI() *cli.App {
 					Aliases: []string{"svc"},
 					Value:   cli.NewStringSlice(temporal.DefaultServices...),
 					Usage:   "service(s) to start",
+					EnvVars: []string{"SERVICES"},
```

Comment:
All of the other env vars are prefixed with `TEMPORAL_`, I would keep that consistency.
I also think that you need to manually split the string now.

Reply:
Ah, because urfave is broken, right!